library/network/src/modules/SuSEFirewall.rb

```diff
@@ -1526,6 +1526,85 @@ def AddSpecialInterfaceIntoZone(interface, zone)
       AddInterfaceIntoZone(interface, zone)
     end
 
+    # Function returns actual state of logging.
+    #
+    # @param [String] unused (for SF2 compatibility)
```

Comment:
If we look at [how this part of the API is actually used](https://github.com/yast/yast-firewall/search?utf8=%E2%9C%93&q=SetLoggingSettings+GetLoggingSettings&type=Code) we see that it expects the SF2 specific values ACCEPT or DROP for the rule, and ALL, CRIT, or NONE for the logging level.

So I am afraid this cannot be ported in a simple way like this.